bot/utils/regex.py

```diff
@@ -7,7 +7,7 @@
     r"discord(?:[\.,]|dot)me|"                        # or discord.me
     r"discord(?:[\.,]|dot)li|"                        # or discord.li
     r"discord(?:[\.,]|dot)io|"                        # or discord.io.
-    r"(?:[\.,]|dot)gg"                                # or .gg/
+    r"(?:\b([\.,]|dot))gg"                                # or .gg/
```

Comment:
```suggestion
    r"(?:\b(?:[\.,]|dot))gg"                                # or .gg/
```
We want this to be a non-capturing group, so that it doesn't affect us extracting the actual invite code.

Also, is this trying to enforce that there is whitespace before the `.gg/`? If so, don't we need to use `\B`?

Reply:
`\b` is a word boundary, `\B` is the opposite.

Reply:
The reason this isn't working is because `.` is not a word character, so it makes no sense to use `\b` here. It's possible to instead have a negative lookbehind for `\w`